### PR TITLE
Assign temp var to open ai models

### LINKED ISSUE #328: BaseOnsiteLLM.generate() got an unexpected keyword argument 'temperature'
running into following error when running local endpoint quickstart example

```
Using model: bloom
Running with an empty context
Exception in thread Thread-2 (new_thread):
Traceback (most recent call last):
  File "/home/codespace/.python/current/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/home/codespace/.python/current/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/home/codespace/.python/current/lib/python3.10/site-packages/llm_vm/completion/optimize.py", line 43, in new_thread
    t[0] = foo()
  File "/home/codespace/.python/current/lib/python3.10/site-packages/llm_vm/completion/optimize.py", line 254, in promiseCompletion
    best_completion = self.call_big(prompt, **kwargs)
  File "/home/codespace/.python/current/lib/python3.10/site-packages/llm_vm/client.py", line 96, in CALL_BIG
    return self.teacher.generate(prompt, max_len,**kwargs)
TypeError: BaseOnsiteLLM.generate() got an unexpected keyword argument 'temperature'
{'status': 0, 'resp': 'cannot unpack non-iterable NoneType object'}
```

### PR DESCRIPTION
**Pull request by Jobhdez**
this fixes #328.

I was able to run the source of error:

```python
>>> from client import Client 
>>> client = Client(big_model='bloom')
`bigscience/bloom-560m` loaded on cuda:0.
`EleutherAI/pythia-70m-deduped` loaded on cuda:0.
Using model: bloom
>>> re = client.complete(prompt = 'What is Anarchy?', context='')
Running with an empty context
>>> re
{'completion': '”\nThe question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is not new. The question is', 'status': 200}
```

### CODE REVIEW

**Comment by Jobhdez** [comment]
@VictorOdede can you review this please if you have time? I also have three others. Thqnks

File: src/llm_vm/onsite_llm.py
Comment (VictorOdede) [inline src/llm_vm/onsite_llm.py:1]: we use ctransformers for running hardware-agnostic quantized llms

**Review by VictorOdede — CHANGES_REQUESTED** [review]

File: src/llm_vm/onsite_llm.py
Comment (Jobhdez) [reply src/llm_vm/onsite_llm.py:1]: > we use ctransformers for running hardware-agnostic quantized llms

I have addressed your review @VictorOdede

**Comment by Jobhdez** [comment]
@VictorOdede Hey, lets merge this if you have time.

**Review by VictorOdede — APPROVED** [review]
Looks good!

**Pull request merged**